internal/config/oauth.go

```diff
@@ -76,6 +113,8 @@ func CallbackHandler(w http.ResponseWriter, r *http.Request) {
 
 	SetOAuthToken(tok.AccessToken)
 	SetOAuthTokenExpiry(tok.Expiry)
+	SetRefreshToken(tok.Extra("refresh_token").(string))
+	SetOAuthRefreshExpiry(time.Now().Add(time.Hour * 720))
```

Comment:
Can't we get the refresh token expiry time from the JWT? I don't think we want to hardcode this value. If we can't get it from the JWT, then better to just always try to refresh and if it fails then fallback to getting a new token